gallery/visualtests/nxCodeSnippet.js

```diff
@@ -20,11 +20,13 @@ describe('NxCodeSnippet', function() {
   } = setupBrowser('#/pages/Code%20Snippet');
 
   const multipleSnippetSelector = '#nx-code-snippet-simple-example .gallery-example-live',
+      sizingSnippetSelector = '#nx-code-snippet-custom-sizing-example .gallery-example-live',
       complexSnippetExample = '#complex-nx-code-snippet',
       textareaSelector = `${complexSnippetExample} textarea`,
       copyBtnSelector = `${complexSnippetExample} .nx-btn`;
 
   it('looks right including distance to other components', simpleTest(multipleSnippetSelector));
+  it('looks right', simpleTest(sizingSnippetSelector));
```

Comment:
Needs a better name.  How about using anested  `describe` block to narrow down which "it" we're talking about:

```
describe('NxCodeSnippet', function() {
...
  describe('with rows=1', function() {
    it('renders with a one-line textarea', simpleTest(sizingSnippetSelector));
  });
});
```

Reply:
Sounds good. I grouped both the simpleTest()s together inside the parent describe('NxCodeSnippet'), but should the hover and focus tests conducted on complexSnippetExample also be included inside that group?

Reply:
> Sounds good. I grouped both the simpleTest()s together inside the parent describe('NxCodeSnippet'), but should the hover and focus tests conducted on complexSnippetExample also be included inside that group?

Oh you misunderstood my example a little bit.  The outer `describe` was intended to be the top-level one already present in the file.  So I was saying all of the other tests should be left as they are, and the new test should be wrapped in `describe('with rows=1'`

Reply:
> > Sounds good. I grouped both the simpleTest()s together inside the parent describe('NxCodeSnippet'), but should the hover and focus tests conducted on complexSnippetExample also be included inside that group?
> 
> Oh you misunderstood my example a little bit. The outer `describe` was intended to be the top-level one already present in the file. So I was saying all of the other tests should be left as they are, and the new test should be wrapped in `describe('with rows=1'`

Thanks for the clarification. Updated in c69f56c and a813df6